saleor/dashboard/widgets.py

```diff
@@ -15,3 +23,17 @@ def __init__(self, attrs=None):
         self.currency = getattr(settings, 'DEFAULT_CURRENCY')
         widgets = (PriceInput(self.currency), PriceInput(self.currency))
         super(RangeWidget, self).__init__(widgets, attrs)
+
+
+class PhonePrefixWidget(PhoneNumberPrefixWidget):
```

Comment:
We only need to customize the `template_name` property, so this widget should override the `saleor.userprofile.widgets.PhonePrefixWidget` class.